webapp/channels/src/components/channel_view/channel_view.tsx

```diff
@@ -104,6 +104,11 @@ export default class ChannelView extends React.PureComponent<Props, State> {
                 this.props.goToLastViewedChannel();
             }
         }
+
+        // If we're restricting direct messages and the value is not yet set, fetch it
+        if (this.props.canRestrictDirectMessage && this.props.restrictDirectMessage === undefined) {
```

Comment:
can we rename `canRestrictDirectMessage` to `canRestrictDirectOrGroupMessage` to be consistent with backend changes?

Reply:
So this gets weird, because the config setting is also named `RestrictDirectMessage`, even though we want it to restrict any kind of private messaging between users that isn't tied to a team.

The correct thing to do would be to change everything as you suggested, but then we'd also likely want to change the name of the configuration setting as well, which I think would require a configuration migration?

Thoughts on if that's worth it?

Reply:
OKay, in that case its fine the way it is